support/NseUpdaterManager.ts

```diff
@@ -1,13 +1,14 @@
 import * as fs from 'fs';
 import { BUNDLE_SHORT_VERSION_TEMPLATE_REGEX, BUNDLE_VERSION_TEMPLATE_REGEX, GROUP_IDENTIFIER_TEMPLATE_REGEX } from './iosConstants';
+import { OneSignalLog } from './OneSignalLog';
 import { ReaderManager } from './ReaderManager';
 
 const entitlementsFilePath =`${__dirname}/serviceExtensionFiles/OneSignalNotificationServiceExtension.entitlements`;
 const plistFilePath = `${__dirname}/serviceExtensionFiles/OneSignalNotificationServiceExtension-Info.plist`;
 
 const logIfError = (err: NodeJS.ErrnoException | null) => {
   if (err) {
-    console.error("Error updating OneSignal NSE Entitlement File.");
+    OneSignalLog.log("Error updating OneSignal NSE Entitlement File.");
```

Comment:
nit, would be nice to keep this as error so it is easier to spot (normally colors it red).